### PR TITLE
Updated the JSON syntax IA

### LINKED ISSUE #2732: Json Syntax Cheat Sheet: Complete re-phrasing
Many people searching for `json syntax`, `json syntax cheat sheet` etc... Will be looking for a quick example of syntax - and oftentimes be beginners; they are not likely to be looking for this:

screenshot from 2016-03-11 14 33 11

I request that someone *please* re-write this cheat sheet so it reads in a manner suitable for general consumption!

/cc @vCillusion

### PR DESCRIPTION
**Pull request by akanshgulati**
###### It is an update to the old JSON syntax IA.

- [ ] Improvement
    - [ ] Enhancement: **`JSON Syntax: Updated the information and added example approach`**

###### Description of changes

Changed the data types and updated the IA with more examples and access methods.

Fixes #2732 - Better information and noobs can easily understand it?

###### People to notify (@moollaza @pjhampton  @GuiltyDolphin )

---

Instant Answer Page: https://duck.co/ia/view/json_syntax_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @vCillusion

### CODE REVIEW

**Comment by daxtheduck** [comment]
@akanshgulati Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by akanshgulati** [comment]
@moollaza  @GuiltyDolphin  @pjhampton Guys please give me updates on this. Thanks

**Comment by moollaza** [comment]
@vCillusion ping! -- You're the maintainer of the Instant Answer. Any feedback on this PR? Do you approve the changes?

**Comment by moollaza** [comment]
IMO this looks too much like JavaScript. the `var`s are misleading.

We should by describing the object and array syntax and the data types. The examples are good and I think we should have more of those under individual headings to give context.

The data types laid out on JSON.org are pretty good. Similarly, http://www.w3schools.com/json/json_syntax.asp is pretty clear.

**Comment by akanshgulati** [comment]
@moollaza have updated the content of IA. I have tried this time to make it more like JSON content than Javascript. Explained about Key Value pair and then changed the examples of JSON values.

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ DDG-*
 *.sw*
 *~
 share/goodie/meta/metadata.json
+.idea
```
Comment (moollaza) [inline .gitignore]: @akanshgulati can you remove this? A CheatSheet PR won't auto deploy to Beta if there are more than 1 files being modified

**Comment by moollaza** [comment]
@akanshgulati once the `.gitignore` change is removed this can be deployed to beta and evaluated.

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

**Pull request merged**